specs/_features/eip7594/das-core.md

```diff
@@ -197,6 +200,34 @@ def get_data_column_sidecars(signed_block: SignedBeaconBlock,
     return sidecars
 ```
 
+#### `get_extended_sample_count`
+
+```python
+def get_extended_sample_count(allowed_failures: uint64) -> uint64:
+    assert 0 <= allowed_failures <= NUMBER_OF_COLUMNS // 2
+
+    def math_comb(n, k):
+        if not 0 <= k <= n:
+            return 0
+        r = 1
+        for i in range(min(k, n - k)):
+            r = r * (n - i) // (i + 1)
+        return r
+
+    def hypergeom_cdf(k, M, n, N):
+        return sum([math_comb(n, i) * math_comb(M - n, N - i) / math_comb(M, N)
```

Comment:
double-check: can this division be `//` here?
```suggestion
        return sum([math_comb(n, i) * math_comb(M - n, N - i) // math_comb(M, N)
```

Reply:
@cskiraly 

I wrote some test cases here with the suggested change above: https://github.com/ethereum/consensus-specs/commit/fb020456cba20ffa76d5c3566c8240b8206a47f5

Does it make sense?

Reply:
> double-check: can this division be // here?

Those are probability values between 0 and 1, so we can't use integer division here.
Here is the notebook I've used to double-check options https://colab.research.google.com/drive/1bbCP7uPmO4j0LnTcbZ4PoAnVyJ8H1KAl

I don't think floating point precision creates issues in this particular case, since we are using the cdf only to find the number of samples at or below the threshold.